updates.jenkins.io.tf

```diff
@@ -17,6 +17,12 @@ resource "azurerm_storage_account" "updates_jenkins_io" {
   tags = local.default_tags
 }
 
+resource "azurerm_storage_share" "updates_jenkins_io" {
+  name                 = "updates-jenkins-io"
+  storage_account_name = azurerm_storage_account.updates_jenkins_io.name
+  quota                = 2 # Current updates.jenkins.io total size in /www/updates.jenkins.io: 400Mo
```

Comment:
2 Gb looks fine for this one. Note for later: could be nice to have this value synced to the PV/PVC quotas